### PR TITLE
fix: diable ads on ssr

### PR DESCRIPTION
**Pull request by parkjh**
- adding checks to disable ads if neccessary
- setting server side prebid config (copied from testad-config.json from render)
- added new yarn commands that were useful

### CODE REVIEW

File: packages/ad/src/ad.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ class Ad extends Component {
       config,
       contextUrl,
       debug: adConfig.debug || false,
+      disableAds: adConfig.diabled || false,
```
Comment (colincclark) [inline packages/ad/src/ad.js]: typo

**Comment by aronshamash** [comment]
@parkjh Could you add descriptions to your PRs?
Also I'm pretty sure we want to check ads are working in SSR not disable them.

**Comment by colincclark** [comment]
Conceptually, why are we disabling ads?

**Comment by parkjh** [comment]
@colincclark @aronshamash my mistake... no we do not want to disable ads on ssr... here's take 2!

**Pull request closed without merging**